### PR TITLE
SW-12632 - Added Theme.php documentation

### PR DESCRIPTION
**Pull request by tobiasberge**
Added a guide for the usage of the Theme.php features

### CODE REVIEW

**Comment by tobiasberge** [comment]
@tiagojsag Could you please have a look at the Theme.php guide?

File: source/designers-guide/index.html [hunk 1]
```diff
@@ -12,6 +12,7 @@
     <li><a href="{{ site.url }}/designers-guide/smarty/">Getting started with Smarty</a></li>
     <li><a href="{{ site.url }}/designers-guide/less/">Getting started with LESS</a></li>
     <li><a href="{{ site.url }}/designers-guide/javascript-statemanager-and-pluginbase">Getting Started with the statemanager and the jQuery plugin base</a></li>
+    <li><a href="{{ site.url }}/designers-guide/theme-php/">Theme.php</a></li>
```
Comment (tiagojsag) [inline source/designers-guide/index.html]: Can you give this guide a better name? Something more informative

File: source/designers-guide/theme-php/index.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+
+-   [Introduction](#introduction)
+-   [Adding JavaScript and CSS](#adding-javascript-and-css)
+-   [Customizing the theme configuration](#customizing-the-theme-configurat)
+-   [Creating a blank theme configuration](#creating-a-blank-theme-configura)
+-   [Configuration sets](#configuration-sets)
+
+## Introduction
+
+The `Theme.php` is the base file of each shopware theme. It contains the basic information like the name, author, description and the license which are displayed in the Theme Manager:
```
Comment (tiagojsag) [inline source/designers-guide/theme-php/index.md]: Please check namings:
- Shopware (the company name with small S, the product with capital S)
- Javascript/CSS/HTML/other things: please make sure that they are consistent across your document and with previously existing doc pages

File: source/designers-guide/theme-php/index.md [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+        'custom-color-main' => '#ff0000'
+    ));
+
+    $collection->add($set);
+}
+```
+
+After saving the `Theme.php` you should see your configuration set inside the configuration sets window:
+
+![Configuration set](config-set.png)
```
Comment (tiagojsag) [inline source/designers-guide/theme-php/index.md]: This is already defined in the first code block, please remove it.

**Pull request merged**